dist/Client.js

```diff
@@ -146,23 +181,28 @@ class APIClient {
 
 
   async uploadRaster(fileName, rasterName) {
+    // Compute file size
     const stream = createReadStream(fileName);
-    let response, data;
-    response = await this._request( // Get upload URL
-    '/rasters/upload/file/', 'POST', {
+    const stats = statSync(fileName);
+    const fileSizeInBytes = stats['size'];
```

Comment:
Do we need this change ? Upload is working fine for me on master, I feel like fetch is doing this already automatically when passed a stream. Or maybe I'm missing what this is fixing ?

Reply:
I've tried the `training.js` example code and UploadRaster raises a 411: which node-fetch version do you have?

Reply:
Mmmh works for me, both on node-fetch 2.6.1 and 2.6.0 (which is our min requirements). Are you sure it's not an issue that the file is non-existing or something ?

Reply:
My bad, messed up with npm, works...

Reply:
It was not npm though, but the fact that I was testing locally: apparently minio raises a 411 if not PUTting with content-length, I'll investigate (not smth urgent anyway)

Reply:
Ah ok its a minio / gcloud difference. Mmmh ok then maybe we can add it back at some point to be able to test.